ast/policy.go

```diff
@@ -433,6 +430,7 @@ func (mod *Module) String() string {
 		return buf
 	}
 
+	buf := []string{}
 	buf = appendAnnotationStrings(buf, mod.Package)
```

Comment:
You're right, that is strange. I think the reason we have it as a function is because we'd repeat essentially the same logic three times. I'll add it to my list of "things to look at later" - inlining may be error prone; it's probably better to have this as a helper method than an anonymous function.